evergreen.yml

```diff
@@ -215,16 +268,48 @@ functions:
         script:  |
           ${prepare_shell}
           if [ 'windows-32' == '${build_variant}' ]; then
-            sed -i 's@SOFTWARE@'"SOFTWARE\\\\Wow6432Node"'@' setup/cleanup.reg
+            sed -i 's@SOFTWARE@'"SOFTWARE\\\\Wow6432Node"'@' setup/cleanup_driver.reg
           fi
-          reg import "setup\cleanup.reg"
+          reg import "setup\cleanup_driver.reg"
           echo "----- Registry entries after clean-up----"
           reg query "HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\ODBC\ODBCINST.INI\ODBC Drivers" -s 2> nul
           reg query "HKEY_LOCAL_MACHINE\SOFTWARE\ODBC\ODBCINST.INI\ODBC Drivers" -s 2> nul
           reg query "HKEY_LOCAL_MACHINE\SOFTWARE\ODBC\ODBCINST.INI\ADF_ODBC_DRIVER" -s 2> nul
           reg query "HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\ODBC\ODBCINST.INI\ADF_ODBC_DRIVER" -s 2> nul
           echo "-------------------------"
 
+  "clean-up crash dump collection":
+    - command: shell.exec
+      params:
+        shell: bash
+        working_dir: mongosql-odbc-driver
+        script:  |        
+          ${prepare_shell}
+          set +e
+          # See what crash dump has been created. Checking the default path just in case
+          echo "----- Dumps collected stored in $DUMP_PATH ----"
+          ls -lrt $DUMP_PATH
+          echo "-------------------------"
+          echo "----- Dumps collected stored in the default dump folder %LOCALAPPDATA%\CrashDumps ----"
+          ls -lrt /cygdrive/c/Users/Administrator/AppData/Local/CrashDumps
+          echo "-------------------------"
+          echo "----- Registry entries before clean-up ----"
+          reg query "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps"
+          echo "-------------------------"
+          if [ -f "$LOCAL_DUMP_ORIGINAL_REG_VAL" ]; then
+            echo "Resetting initial registry values"
+            cat $LOCAL_DUMP_ORIGINAL_REG_VAL
+            reg import $LOCAL_DUMP_ORIGINAL_REG_VAL
+          else
+            echo "Resetting initial registry values - Deleting entry"
+            reg import setup/cleanup_dumps_collection.reg
+          fi
+          EXITCODE=$?
+          echo "----- Registry entries after clean-up ----"
+          reg query "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps" -s 2> nul
```

Comment:
I see this `2> nul` in other parts of the code as well.  I think we mean to have it be `2> /dev/null` if the intended behavior is to ignore the error in case there is one.  The way it is written would write to a file `nul`